src/data/battle_moves.h

```diff
@@ -8322,15 +8354,19 @@ const struct BattleMove gBattleMoves[MOVES_COUNT] =
 
     [MOVE_STEAMROLLER] =
     {
-        .effect = EFFECT_FLINCH_MINIMIZE_HIT,
+        #if B_UPDATED_MOVE_DATA >= GEN_6
+            .flags = FLAG_MAKES_CONTACT | FLAG_PROTECT_AFFECTED | FLAG_MIRROR_MOVE_AFFECTED | FLAG_SHEER_FORCE_BOOST | FLAG_DMG_MINIMIZE,
+        #else
+            .flags = FLAG_MAKES_CONTACT | FLAG_PROTECT_AFFECTED | FLAG_MIRROR_MOVE_AFFECTED | FLAG_SHEER_FORCE_BOOST,
+        #endif
```

Comment:
I don't think Steamroller should get a config because it was always hitting Minimize doubly since its debut, just like Flying Press which doesn't have a config.

Reply:
mistake on my part